### PR TITLE
Fix statistics

### PR DESCRIPTION
**Pull request by Maikol1804**
All the statistics plots were fixed, so that now the name of the task is shown instead of the ID

### CODE REVIEW

File: inginious/frontend/webapp/plugins/statistics/pages.py [hunk 1]
```diff
@@ -371,10 +409,12 @@ def API_GET(self):
 
         for task in sorted_tasks:
```
Comment (larranaga) [inline inginious/frontend/webapp/plugins/statistics/pages.py]: This code is repeated, could this function be abstracted into a function?

File: inginious/frontend/webapp/plugins/statistics/static/js/user_statistics.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ var UserTrialsAndBestGradeStatistic = (function () {
     };
 
     UserTrialsAndBestGradeStatistic.prototype._plotData = function (tries_per_tasks) {
+        console.log(tries_per_tasks);
```
Comment (larranaga) [inline inginious/frontend/webapp/plugins/statistics/static/js/user_statistics.js]: remove logs please

File: inginious/frontend/webapp/plugins/statistics/pages.py (hunk 1 above)
Comment (Maikol1804) [reply inginious/frontend/webapp/plugins/statistics/pages.py]: Yes, I'm going to abstact this code in a function.

**Review by larranaga — APPROVED** [review]
LGTM

**Pull request merged**